src/core/commandRunner/CommandRunner.ts

```diff
@@ -66,6 +66,9 @@ export default class CommandRunner {
         action.getTargetPreferences(...extraArgs)
       );
 
+      this.graph.debug.log("Full targets:");
```

Comment:
Do you mind adding a (debug.active) around these? Seems unnecessary to run stringify if the result is not going to be used